static/templates/settings/display_settings.hbs

```diff
@@ -47,10 +47,14 @@
             <h3 class="inline-block">{{t "Time settings" }}</h3>
             <div class="alert-notification" id="time-settings-status"></div>
 
-            {{> settings_checkbox
-              setting_name="twenty_four_hour_time"
-              is_checked=page_params.twenty_four_hour_time
-              label=settings_label.twenty_four_hour_time}}
+            <div class="input-group">
+                <label for="twenty_four_hour_time" class="dropdown-title">{{t settings_label.twenty_four_hour_time }}</label>
```

Comment:
First, `t` is used along with a string e.g. `"Time format"`, not a context variable like done above, nonetheless, you should remove this variable anyway from both `admin.js` and `settings.js`, and directly put the string here as we use these variable for checkboxes only (because of translation issue).

Reply:
ok will do thx